### PR TITLE
update readme-v0.3 release

### PR DESCRIPTION
**Pull request by tong-shao**

### CODE REVIEW

**Review by wenxcs — DISMISSED** [review]

**Comment by wenxcs** [comment]
\azp run

File: docs/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Welcome to the NNFusion Documents!
    - [End-to-end-Test](End-to-end-Test.md)
 3. [Compile-a-Tensorflow-model-with-NNFusion](Compile-a-Tensorflow-model-with-NNFusion.md)
    - [Compile-a-model-with-kernel-tuning-enabled](Compile-a-model-with-kernel-tuning-enabled.md)
+   - [How to use NNFusion Python interface for inference/training](../src/python/example/README.md)
```
Comment (xysmlx) [inline docs/README.md:12]: This is for PyTorch. So, maybe the position or the higher level title needs modification.

Comment (wenxcs) [reply docs/README.md:12]: Yes, but what's more suitable

Comment (xysmlx) [reply docs/README.md:12]: Change the higher level title as "Compile a model with NNFusion", move "Compile-a-Tensorflow-model-with-NNFusion.md" to lower level? The PyTorch/ONNX frontend also needs a doc?

Comment (wenxcs) [reply docs/README.md:12]: Great

**Review by wenxcs — APPROVED** [review]

**Pull request merged**